cli/main.go

```diff
@@ -55,6 +57,34 @@ func main() {
 	app.Commands = append(app.Commands, (&commands.Watch{}).Commands()...)
 	app.Commands = append(app.Commands, (&commands.Project{}).Commands()...)
 	app.Commands = append(app.Commands, (&commands.Doctor{}).Commands()...)
+	app.Commands = append(app.Commands, (&commands.Noop{}).Commands()...)
+
+	notificationTitle := fmt.Sprintf("Outrigger CLI (rig) v%s", VERSION)
+	notify.Init(app.Name, notificationTitle, "images/logo.png", map[string]bool{
```

Comment:
Should we let the commands themselves contain the config for notification status?

Reply:
Yes, I didn't understand how to modify the command struct until the last few days. We should definitely move this.